tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/MsTestV2/TestAssemblyInfoRunAssemblyInitializeIntegration.cs

```diff
@@ -83,4 +93,11 @@ internal static CallTargetReturn OnMethodEnd<TTarget>(TTarget instance, Exceptio
     private static void EmptyCleanUpMethod()
     {
     }
+
+#pragma warning disable SA1201
+    internal interface ITestAssemblyInfoWithAssembly : ITestAssemblyInfo
```

Comment:
Is there any value in deriving from `ITestAssemblyInfo` here? You're only accessing the `Assembly` - doesn't that mean you're doing more work mapping other members you're never going to use?

Reply:
you're totally right! done in https://github.com/DataDog/dd-trace-dotnet/pull/6686/commits/a565f8452c22f2fabbeaf9e33fad85bd090faa55